pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -23,7 +23,7 @@ class PacketOut(GenericMessage):
     #: :class:`~pyof.v0x01.common.header.Header`
     header = Header(message_type=Type.OFPT_PACKET_OUT)
     buffer_id = UBInt32()
-    in_port = UBInt16()
+    in_port = UBInt16(enum_ref=Port)
```

Comment:
in_port should not have an enum_ref, as you said in https://github.com/kytos/python-openflow/issues/364#issuecomment-300531170